lib/ansible/modules/windows/win_pester.py

```diff
@@ -32,6 +32,11 @@
       - Accepts multiple comma seperated tags.
     type: list
     version_added: '2.9'
+  parameterlist:
```

Comment:
Looks like this was missed in the recent commit, should be `test_parameters`.

Reply:
yeah, I missed it